alternator/server.cc

```diff
@@ -231,7 +231,19 @@ future<json::json_return_type> server::handle_api_request(std::unique_ptr<reques
 }
 
 void server::set_routes(routes& r) {
-    _callbacks = {
+    api_handler* handler = new api_handler([this] (std::unique_ptr<request> req) mutable {
+        return handle_api_request(std::move(req));
+    });
+
+    r.add(operation_type::POST, url("/"), handler);
+}
+
+//FIXME: A way to immediately invalidate the cache should be considered,
+// e.g. when the system table which stores the keys is changed.
+// For now, this propagation may take up to 1 minute.
+server::server(seastar::sharded<executor>& e)
+        : _executor(e), _key_cache(1024, 1min, slogger), _enforce_authorization(false)
+      , _callbacks{
         {"CreateTable", [] (executor& e, executor::client_state& client_state, std::unique_ptr<request> req) {
```

Comment:
just a comment, no changes requested: now we could actually capture `this` in the lambda and omit the `executor` parameter, but it's not really relevant for the performance and perhaps it will even be less readable than it currently is.

Reply:
I think I can make it more readable by DRYing the repetitive code substantially.  But that's for later, after the fix goes in...